install/amazon_rds/05_configure_the_collector_ec2.mdx

```diff
@@ -19,22 +19,27 @@ export const APIKeyInstructions = ({ apiKey }) => {
   );
 };
 
+export const ConfigSample = ({ apiKey }) => {
+  return (
+    <CodeBlock>
+      {`[pganalyze]
+api_key = ${apiKey ?? 'your_pga_organization_api_key'}\n
+[server1]
+db_host = instance-id.account-id.us-east-1.rds.amazonaws.com
+db_name = your_database_name
+db_username = your_monitoring_user
+db_password = your_monitoring_user_password
+db_sslrootcert = rds-ca-2019-root
+db_sslmode = verify-full`}
+    </CodeBlock>
+  )
+}
+
 ## Configuring the collector on an Amazon EC2 instance
 
 The collector configuration file lives in `/etc/pganalyze-collector.conf`, and looks like this:
 
-```
-[pganalyze]
-api_key: your_pga_organization_api_key
-
-[server]
-db_host: instance-id.account-id.us-east-1.rds.amazonaws.com
-db_name: your_database_name
-db_username: your_monitoring_user
-db_password: your_monitoring_user_password
-db_sslrootcert: rds-ca-2019-root
-db_sslmode: verify-full
-```
+<ConfigSample apiKey={props.apiKey} />
 
 **Fill in the values step-by-step:**
```

Comment:
We should probably have that on the last step on all install environments, right? Any thoughts on copy? Something like

>We recommend setting up <a href="/docs/log-insights/setup">Log Insights</a> as a follow-up step, to automatically track log events in your database?

?

Reply:
Yep, I think that works - or it could also be a button that says "Continue with Log Insights setup"

Reply:
I think these would work better as links to avoid a uh "CTA treadmill" as part of setup. That is, we're using the buttons to indicate a series of steps within a phase of setup, and linking across phases with the same style might be confusing, especially since in the public docs we can't verify that you managed to complete the phase before we show you the link.